tsconfig.json

```
@@ -14,7 +14,10 @@
     "isolatedModules": true,
     "jsx": "preserve",
     "incremental": true,
-    "baseUrl": "src"
+    "baseUrl": "src",
+    "paths": {
+      "@/styles/*": ["styles/*"]
```

それでも間違ってないんですが、

```suggestion
    "baseUrl": ".",
    "paths": {
      "@/*": ["./src/*"]
    }
```

こっちの方が `styles` 以外のディレクトリにも使えて便利です。